packages/lodestar/src/chain/chain.ts

```diff
@@ -326,6 +328,7 @@ export class BeaconChain extends (EventEmitter as { new(): ChainEventEmitter })
     // add justified block to forkchoice so "this.justified" in forkchoice really map to a block
     if (isStateNotGenesis) {
       const preJustifiedBlock = await this.db.blockArchive.getByRoot(lastKnownState.currentJustifiedCheckpoint.root);
+      if (!preJustifiedBlock) throw Error("lastKnownState not in db");
```

Comment:
Many error messages are raw, feel free to propose better more descriptive messages where necessary.

Reply:
if you got this far in code, you already have lastKnwon state so message should be lastJustifiedBlock not in database